packages/forms-web-app/src/apis/appealsApi.js

```diff
@@ -0,0 +1,58 @@
+const axios = require('axios');
+const uuid = require('uuid');
+const config = require('../config');
+const parentLogger = require('../lib/logger');
+
+const send = async (method, params = {}) => {
+  const correlationId = uuid.v4();
+  const url = `${config.appeals.url}${params.url}`;
+  const logger = parentLogger.child({
+    correlationId,
+    service: 'Appeals Service API',
+  });
+
+  try {
+    logger.debug({ method, params }, 'New Appeals service request');
+
+    const apiResponse = await axios.request({
+      url,
+      method,
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Correlation-ID': correlationId,
+        ...params.headers,
+      },
+    });
+
+    logger.debug('Api successfully responded with', apiResponse);
```

Comment:
If given, the logger statements should log the object as the first param.

https://github.com/pinojs/pino/blob/master/docs/api.md#errors

`logger.debug(apiResponse, 'Api successfully responded with');`

or

`logger.debug({ apiResponse, otherInterestingObjects }, 'Api successfully responded with');`

Reply:
nice catch ... cheers